### DISCUSSION TITLE
Source Control Issue on VScode

### DISCUSSION THREAD

**Discussion by 0xkriswebchain** [Cyfrin/foundry-full-course-cu#1409] [General]
Hello, in my nft project, I have not more than 12 files to push on github repository, but when I check source control it showing more than 60 files want to commit change and push to git repository that are not event part of the project. Why it shows like that, how shall I avoid them.

**Comment by heheboii11** [comment]
Just do 
```git commit -m "initial commit"```
and then 
```git remote add origin link of the git repo you created```
and then 
```git push -u origin main```
check in the second line you have to give your repo link
